index.html

```diff
@@ -26,22 +28,151 @@
     <!-- Headline Section Start -->
     <section class="headline">
     <div class="container_text">
-        <h1 class="ptext">Hey There. <br/>I'm Zuwaira.</h1>
+        <h1 class="ptext">Hey There. <br/>I'm Zuwairanajma.</h1>
+
         <h3 class="stext">I'm a Software Developer</h3>
         <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hesitate to contact me.</p>
             <nav class="icon-wrap">
                 <ul class="navicon">
-                   <li><a href="https://github.com/Zuwairanajma"><img class="github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
-                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class="linkedin" src ="PImages/Linkedin.png" alt = "Linkedin Icon"></a></li> 
-                   <li><a href="#"><img class ="angeli" src ="PImages/Vector3.png" alt="Angellist Icon"></a></li>
-                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class ="twitter" src ="PImages/Twitter.png" alt ="Twitter Icon"></a></li>
-                   <li><a href="#"><img class ="medium" src="PImages/Vector5.png" alt ="Medium Icon"></a></li>
+                   <li><a href="https://github.com/Zuwairanajma"><img class= "github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
+                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class= "linkedin"src="PImages/Linkedin.png" alt="Linkedin Icon"></a></li> 
+                   <li><a href="#"><img class= "angeli"src="PImages/VectorH.png" alt="Angellist Icon"></a> </li>
+                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class= "twitter" src="PImages/Twitter.png" alt="Twitter Icon"></a></li>
+                   <li><a href="#"><img class= "medium" src="PImages/Medium.png" alt="Medium Icon"></a></li>
+
                 </ul>
             </nav>
         </div>
     </section>
     <!-- Headline Section End -->
+  
+    <!-- Recent Works Section Start -->
+    <section id="My-Recent" class="My-Recent">
+        <div id="recent-works_container" class="recent-works">
+        <h1 id="my-work" class="my-work">My Recent Works</h1>
```

Comment:
- [ ] Kindly consider changing this from h1 to any other `h2 to h6` because there should be only one h1 in our webpage for SEO and semantics purposes.